src/api-config.js

```diff
@@ -36,4 +36,9 @@ export const endpoints = {
     student: userId => `${baseUrl}/students/${userId}/alfon`,
     group: groupId => `${baseUrl}/groups/${groupId}/alfon`,
   },
+
+  onlineLessons: {
+    all: `${baseUrl}/bbb`,
```

Comment:
Is this the actual URL? 😅

Reply:
Yes 😂
Actually it's an abbreviation of [BigBlueButton](https://github.com/bigbluebutton/bigbluebutton)